### DISCUSSION TITLE
2 camera spincapture with ffmpeg video recording?

### DISCUSSION THREAD

**Discussion by dlhagger** [bonsai-rx/bonsai#1726] [Q&A]
I am attempting to set up a dual camera capture for spinnaker cameras that grabs the nearest frames from each other and then writes the files to video. Currently I am using the video writer node, but is there a way to do this with ffmpeg and possibly use a hardware accelerated encoder (hevc-evenc) to allow for tuning on the compression and speed? These are mono cameras and the video recordings are relatively long (~24 hours). I have attached my current workflow. Any help is appreciated!

David

[mousehouse_2caM.bonsai.zip](https://github.com/bonsai-rx/bonsai/files/14856272/mousehouse_2caM.bonsai.zip)

**Comment by bruno-f-cruz** [comment]
Hi!

Have you tried the VideoWriter from the FFMPEG package?
I have used this ffmpeg string in the past (not saying is the best, just something that worked :P)
`-c:v h264_nvenc -vsync 0 -2pass 0 -bf:v 0 -qp 24 -preset llhp -b:v 6M -rc:v cbr`
